homeworks/hw2/stackCalculator/StackCalculatorTests/PostfixCalculatorTests.cs

```diff
@@ -0,0 +1,58 @@
+namespace StackCalculatorTests;
+
+public class PostfixCalculatorTests
+{
+    private StackOnLinkedList stack;
```

Comment:
nullability-анализу это не должно нравиться, поле null после конструктора. Правильнее 
```suggestion
    private StackOnLinkedList stack = new();
```